api/feature-flags.js

```
@@ -0,0 +1,67 @@
+const express = require('express');
+const router = express.Router();
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)